### PR TITLE
Update rubygem-foreman_inventory_upload to 2.0.3

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by laviro** [comment]
[test rpm]

**Comment by laviro** [comment]
Seems that npm devDependencies are failing, no idea why..
cc @sharvit @evgeni 

```
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/eslint-plugin-foreman) < 5.0.0
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/eslint-plugin-foreman) >= 4.0.2
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/stories) < 5.0.0
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/stories) >= 4.0.2
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/test) < 5.0.0
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/test) >= 4.0.2
DEBUG util.py:744:  Child return code was: 1
```

File: packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec [hunk 1]
```diff
@@ -35,71 +35,25 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
...
+BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/eslint-plugin-foreman) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/eslint-plugin-foreman) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) < 5.0.0
```
Comment (sharvit) [inline packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: AFAIK they should be ignored

Comment (laviro) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: should we erase those lines? 
or is it a bug in `koji` which should ignore `devDependencies`

Comment (evgeni) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: If these are not needed for building, they should be added to the list in https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L10

Comment (laviro) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: actually they are listed there,
maybe I am using an old `gem2rpm` version.. I will try again

Comment (ekohl) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: Typically it's a sign you need to rebase on a more recent `rpm/develop`.

**Comment by zjhuntin** [comment]
[test rpm]

**Pull request closed without merging**